.github/workflows/changes.yml

```diff
@@ -0,0 +1,30 @@
+name: changes
+
+on: [pull_request, push]
+
+jobs:
+  run_changes:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout PR
+        uses: actions/checkout@v2
+
+      - name: Set up ocaml
+        uses: avsm/setup-ocaml@v1
+        with:
+          ocaml-version: 4.07.1
+
+      - name: Set up latest ocaml
+        run: |
+          opam switch install 4.12.0
+          opam install --switch 4.12.0 opam-dune-lint
+          eval $(opam env)
+
+      - name: Make sure we can build
+        run: |
+          opam install . --deps-only
+
+      - name: Compare opam and dune files
+        run: |
+          opam exec --switch 4.12.0 opam-dune-lint 
```

Comment:
I use 4.07.1 to build, but I use 4.12.0 to run opam-dune-lint, it seems to work

Reply:
If it doesn't work, can you remove it? I don't think we want to add failing CI jobs.